code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -21,6 +21,11 @@
 	var/core_name = "hypospray"
 	var/label = null
 
+/obj/item/reagent_container/hypospray/advanced
+	name = "\improper advanced hypospray"
+	desc = "The hypospray is a sterile, air-needle reusable autoinjector for rapid administration of drugs to patients with customizable dosages. Comes complete with an internal reagent analyzer and digital labeler. Handy."
+	var/core_name = "hypospray"
```

Comment:
the `var/` prefix makes this not compile. It's already defined by the parent type.